RiGolo/rigolo-0001.adoc

```diff
@@ -0,0 +1,105 @@
+API. The RiGolo should provide a concise technical specification, as
+well as a rationale and motivation.
+
+The main purpose of the RiGolo is to collect community input and
+document the design decisions. The author is responsible for building
+consensus within the community and documenting dissenting opinions,
+which could be lost if Golo ever changes its hosting platform.
+
+Because the RiGolos are maintained in a versioned repository, their
+revision history is the historical record of the proposal.
+
+== RiGolo Workflow
+
+=== Golo BDFL
+
+This acronym stands for “Benevolent Dictator for Life” and refers to
+Julien Ponge https://github.com/jponge[@jponge], the original creator
+of, and the final design authority for, the Golo programming language.
+
+=== RiGolo or Issue?
+
+It is highly recommended that a RiGolo contain a single key proposal or
+new idea. Small enhancements or patches often don't need a RiGolo and
+can be proposed by opening an issue in the golo-lang
+https://github.com/eclipse/golo-lang/issues[issue tracker] or by
+directly opening a pull request for discussion.
+
+The more focused the RiGolo, the more successful it tends to be. If in
+doubt, split your RiGolo into several well-focused ones.
+
+=== Submitting a RiGolo
+
+Each RiGolo must have a main author responsible for building community
+consensus around the idea. The main author should also determine whether
+the idea deserve a RiGolo or not. Starting a preliminary issue in the
+https://github.com/eclipse/golo-lang/issues[tracker] or posting a
+message on the mailto:golo-dev@eclipse.org[golo-dev mailing list] are
+the best way to go about this. Asking the Golo community first if an
+idea is original helps prevent too much time being spent on something
+that is guaranteed to be rejected based on prior discussions. It also
+helps to make sure the idea is applicable to the entire community and
+not just the author.
+
+According to this first feedback, a draft RiGolo will be proposed as a
+pull request on the `golo-lang/rigolo` repository. This draft must take
+the form of a new file in the `RiGolo` directory, with the format and
+style described link:#rigolo-formats-and-structure[bellow], and a “Draft”
+status.
+
+=== RiGolo Review & Resolution
+
+The draft can thus be discussed further in the pull request comments,
+and changes can be done by pushing new commits to the pull request. It
+is the responsibility of the main author to keep the draft up to date
+with the feedback from discussions and reviews.
+
+If no consensus can be reach, the BDFL will decide.
+
+When the draft reaches a satisfactory state, a RiGolo number is given
+and the pull request can be merged. For a RiGolo to be accepted it must
+be a clear and complete description of the proposed improvement. It must
+moreover comply with the Golo philosophy, which is whatever is
+acceptable by the BDFL.
+
+A RiGolo can also be “Rejected”, and the pull request closed. Perhaps
```

Comment:
Should the PR be closed, or merged with a “Rejected” status, to keep the record independent of the code platform but in the repository itself?